shell/common/node_bindings.cc

```diff
@@ -440,6 +440,15 @@ node::Environment* NodeBindings::CreateEnvironment(
   if (browser_env_ != BrowserEnvironment::kBrowser)
     global.Set("_noBrowserGlobals", true);
 
+  if (browser_env_ == BrowserEnvironment::kBrowser) {
+    const std::vector<std::string> search_paths = {"app.asar", "app",
+                                                   "default_app.asar"};
+    const std::vector<std::string> app_asar_search_paths = {"app.asar"};
+    global.Set("_appSearchPaths", electron::fuses::IsNonAsarAppLoadingEnabled()
```

Comment:
Can we set a private instead of using `delete`?

Reply:
🤷  Guess so, I don't think there is any difference given we delete and this array isn't secret anyway

Reply:
Mainly `delete` feels a bit gross as a cleanup operation, occasionally something goes wrong and the bit of code that `delete`s is never called. Probably not an issue in this case, but using a private is fairly easy anyway.